misc.php

```diff
@@ -275,6 +275,12 @@
 	if ($pun_config['o_censoring'] == '1')
 		$cur_post['subject'] = censor_words($cur_post['subject']);
 
+	$crumbs = generate_crumbs(array(
+		array($lang_common['Index'], 'index.php'),
+		array(pun_htmlspecialchars($cur_post['forum_name']), 'viewforum.php?id='.$cur_post['fid']),
+		array(pun_htmlspecialchars($cur_post['subject']), 'viewtopic.php?pid='.$post_id.'#p'.$post_id),
```

Comment:
What about moving the `pun_htmlspecialchars` calls to the function, too?

Reply:
I did it like the following where some entries require it and others don't like language strings.

`$page_title = array(pun_htmlspecialchars($pun_config['o_board_title']), $lang_common['User list']);`

Reply:
Yeah, I saw that. But it won't be a problem with language strings - and I like APIs that do the safe thing for you automatically, which means it cannot be forgotten.,

Reply:
Sounds good! Will do.

Reply:
Done.

Reply:
Aargh. It is not going to work with Search results.
```
    Index » Search » <a href="search.php?action=show_recent">Active</a>
```